dask/array/tests/test_fft.py

```diff
@@ -137,6 +137,7 @@ def test_wrap_ffts(modname, funcname, dtype):
 
     wfunc = fft_wrap(func)
     assert wfunc(darrc).dtype == func(nparrc).dtype
+    assert wfunc(darrc).shape == func(nparrc).shape
```

Comment:
Ah sorry GitHub hid your previous comment. Well one thing to consider (and this might be the right way to fix it) is SciPy and NumPy provide different return types (real or complex). So the chunking function could take the return type as an argument and infer what to do.